src/test/suite/tokenizer.test.ts

```diff
@@ -60,13 +67,47 @@ const tests: TestCase[] = [
   ["aåäöb", ["aåäöb"]],
 ];
 
+const languageTokenizerTests: LanguageTokenizerTests = {
+  css: {
+    tests: [
+      ["min-height", ["min-height"]],
+      ["-webkit-font-smoothing", ["-webkit-font-smoothing"]],
+      ["(min-width: 400px)", ["(", "min-width", ":", "400px", ")"]],
+      ["prefers-reduced-motion", ["prefers-reduced-motion"]],
+    ],
+    // Leave kebab and dashes to css language specific tests.
+    exclusionPredicate: (input: string) => input.match("-") == null,
+  },
+};
+
 suite("tokenizer", () => {
   tests.forEach(([input, expectedOutput]) => {
-    test(input, () => {
-      const output = tokenize(input, (match) => match[0]);
+    test(`tokenizer test, input: ${input}`, () => {
+      const output = tokenize(input, "anyLang", (match) => match[0]);
       assert.deepStrictEqual(output, expectedOutput);
     });
   });
+
+  Object.entries(languageTokenizerTests).forEach(
+    ([language, { tests, exclusionPredicate = () => true }]) => {
+      tests.forEach(([input, expectedOutput]) => {
+        test(`${language} custom tokenizer, input: ${input}`, () => {
+          const output = tokenize(input, language, (match) => match[0]);
+          assert.deepStrictEqual(output, expectedOutput);
+        });
+      });
+
+      tests.forEach(([input, expectedOutput]) => {
```

Comment:
I believe you're intending to use the `tests` variable from the global scope here, but it is being shadowed by the `tests` parameter